prql/src/parser.rs

```diff
@@ -207,13 +207,12 @@ fn ast_of_parse_tree(pairs: Pairs<Rule>) -> Result<Vec<Node>> {
                 _ => unreachable!(),
             };
 
-            Ok(Node {
-                item,
-                span: Span {
-                    start: span.start(),
-                    end: span.end(),
-                },
-            })
+            let mut node = Node::from(item);
```

Comment:
(FYI there's a way to do this natively in rust: https://doc.rust-lang.org/book/ch05-01-defining-structs.html#creating-instances-from-other-instances-with-struct-update-syntax
...but exactly the same result)